<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SolarusEditor::OpenQuestFileDialog</class>
 <widget class="QDialog" name="SolarusEditor::OpenQuestFileDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>189</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Open quest file</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <widget class="LineEdit" name="search_field"/>
   </item>
   <item>
    <widget class="QListWidget" name="list_widget"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>widgets/line_edit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
